### DISCUSSION TITLE
Run example read format h.264

### DISCUSSION THREAD

**Discussion by dotrungson** [bluenviron/gortsplib#449] [Questions] [answered]
Hi everyone,

I'm new in Golang and gortsplib, I want to create a client that reads RTSP stream and extract Frames using Golang. I try to run example client-read-format-h264 but it throw error about undefined funtion :(

![image](https://github.com/bluenviron/gortsplib/assets/17586977/ae6bc20c-f244-4f8c-bb49-8c78c60f47d7)

Somebody can help me fix this issue.

**Comment by aler9** [comment, marked as answer]
Hello, JetBrains executed this (wrong) command:

```
go build examples/client-read-format-h264/main.go
```

Try compiling the folder instead, either by configuring JetBrains (i don't know how to do it since i don't use it) or by running this command manually:

```
go build examples/client-read-format-h264
```

Be aware that the `client-read-format-h264` also requires the FFmpeg libraries, which are not available on plain Windows.

**Reply by dotrungson** [reply]
Thank you so much for answer, @aler9
